cpp/src/arrow/engine/substrait/serde.h

```diff
@@ -133,8 +132,7 @@ ARROW_ENGINE_EXPORT Result<std::vector<acero::Declaration>> DeserializePlans(
 /// \param[out] ext_set_out if non-null, the extension mapping used by the Substrait
 /// Plan is returned here.
 /// \param[in] conversion_options options to control how the conversion is to be done.
-/// \return a vector of ExecNode declarations, one for each toplevel relation in the
-/// Substrait Plan
+/// \return an ExecPlan for the Substrait Plan
```

Comment:
Revert this?

Reply:
The pre-PR comment is incorrect, so I fixed it.

Reply:
Oh you are right I am looking at the wrong thing